incubator/fluentd/files/papertrail.conf

```diff
@@ -29,9 +29,11 @@
 <match **>
   @type papertrail
   @id out_papertrail
-  num_threads 4
-  buffer_type    file
-  buffer_path    /var/log/fluentd-buffers-papertrail/fluentd-buffer
+  <buffer>
+    @type    file
+    path    /var/log/fluentd-buffers-papertrail/fluentd-buffer
+    flush_thread_count 4
```

Comment:
Is there a reason for `flush_thread_count` to be hard-coded here, but not in the elasticsearch.conf file? Or could this have a templated value set from `values.yaml` as well?